<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label_Id">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Pwd">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>密  码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Pwd">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Login">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>190</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Regist">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>190</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="pushButton_Back_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>230</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Pwd_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>密  码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Pwd_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Regist_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>230</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Id_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_Id_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_PwdSure_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>确认密码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_PwdSure_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>130</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Check_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_Check_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>验证码：</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
